packages/react-flow-editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kseniass/react-flow-editor",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
it's not necessary update version, it's doesn't contain any changes for lib users